src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -13,10 +13,10 @@ import scala.language.implicitConversions
 
 class DnsRecordClient(executor: CloudflareApiExecutor)(implicit val ec: ExecutionContext) {
 
-  import Implicits._
+  import com.dwolla.cloudflare.domain.model.Implicits._
 
-  protected lazy val logger: Logger = LoggerFactory.getLogger("LambdaLogger")
-  protected implicit val formats: Formats = DefaultFormats ++ org.json4s.ext.JodaTimeSerializers.all
+  protected lazy val logger: Logger = LoggerFactory.getLogger(classOf[DnsRecordClient])
```

Comment:
I noticed the `logger` variable isn't even used in the portion I pulled out. You mind if I just remove the reference?